roles/debug_tools/files/podman-print-secret

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+podman secret inspect --showsecret --format '{{.SecretData}}' $1
```

Comment:
Perhaps quote it just to be sure?
```suggestion
podman secret inspect --showsecret --format '{{.SecretData}}' "$1"
```

It could also be an alias in the bash config, even though that may be harder to manage.
```
alias podman-print-secret="podman secret inspect --showsecret --format '{{.SecretData}}'"
```